src/assets/ElmAsset.js

```diff
@@ -0,0 +1,43 @@
+const JSAsset = require('./JSAsset');
+const localRequire = require('../utils/localRequire');
+const process = require('process');
+
+// Currently no pretty way to install Elm tools on behalf of the user :/
+async function compile(name, options) {
+  try {
+    const elmCompiler = await localRequire('node-elm-compiler', name);
+    // compileToStringSync does not throw an Error, even when there is one.
+    // Other methods will throw an error, but do not bubble up to Parcel
+    const data = elmCompiler.compileToStringSync(name, options);
+    if (!data) {
+      throw 'You either have an error in your code, or are missing the Elm tools here https://guide.elm-lang.org/install.html.';
```

Comment:
Would be nice to have an actual error if possible from the compiler... you mentioned something about that in your comment. What issue did you have?

Either way, don't throw a string. Throw an Error object, for stack traces. `throw new Error('...')`

Reply:
Yep, have this fixed, need to push my latest. The elm compilers errors will still make it through to the console, but there is a case where a missing dependency does not bubble up an error :/